nflow-tests/src/main/java/io/nflow/tests/demo/workflow/TestCronWorkflow.java

```diff
@@ -0,0 +1,21 @@
+package io.nflow.tests.demo.workflow;
+
+import io.nflow.engine.workflow.curated.CronWorkflow;
+import io.nflow.engine.workflow.definition.NextAction;
+import io.nflow.engine.workflow.definition.StateExecution;
+import org.springframework.stereotype.Component;
+
+import static io.nflow.engine.workflow.curated.CronWorkflow.State.schedule;
+import static io.nflow.engine.workflow.definition.NextAction.moveToState;
+
+@Component
+public class TestCronWorkflow extends CronWorkflow {
```

Comment:
It would be nice to show here all the things that can be overridden (handleFailure, settings, getNextActivation)

Reply:
Or maybe one SimpleCron example and then another one for showing all the moving parts

Reply:
This just tests that the cron workflow works. I think we have examples separately?

Reply:
I think the nflow-tests package is the best place for working examples as they are continuously tested and thus easy to keep up to date when things change.